internal/runtime/kubernetes/kubeobserver/waiter.go

```diff
@@ -42,24 +41,23 @@ func WaitForCondition[Client any](ctx context.Context, cli Client, action *tasks
 	})
 }
 
-func PrepareEvent(gvk kubeschema.GroupVersionKind, namespace, name, desc string, deployable runtime.Deployable) *orchestration.Event {
+func PrepareEvent(gvk kubeschema.GroupVersionKind, namespace, name, desc string, scope string) *orchestration.Event {
```

Comment:
Why change the signature? Isn't it clearer to either receive a deployable or not. Can still use this method to prepare an event for a non-deployable.

Reply:
I can leave the signature as is and set the scope by hand later.

I mostly want to set a scope in general, since the fallback label that is printed is the resource ID, which is too k8s specific 

See:
![image](https://user-images.githubusercontent.com/2537713/199525403-3ad5ab6b-5f50-4108-91f1-bfa1cd76748f.png)